torch/csrc/jit/script/compiler.h

```diff
@@ -233,6 +233,8 @@ TORCH_API Value* emitBuiltinCall(
   // otherwise it will return nullptr if the builtin is not found.
   bool required);
 
+TORCH_API TypePtr parseTypeFromExpr(Expr expr);
```

Comment:
This isn't needed outside of `compiler.cpp` is it? If so it should stay as a forward declaration in there instead of out here in the header

Reply:
Yeah looks like only the compiler is using parseTypeFromExpr, I will update.